src/domain/proof-of-liabilities/generate-leaves.ts

```diff
@@ -0,0 +1,19 @@
+import { createHash } from "crypto"
```

Comment:
I know that probably you want to test the concept first but doing the library will help you to design a better API. you are mixing  responsabilities